src/views/partials/partners/partner.hbs

```diff
@@ -1,5 +1,7 @@
 {{#if img}}
 <div class="col-sm-4 align-self-center">
-  <img src="{{img}}" class="partner-img" alt="{{name}}" />
+  <a href="{{link}}" target="_blank" rel="noopener">
```

Comment:
Nice one! 👍

Reply:
Please, run it also in WebAIM/Chrome Audit, I think they might not like that there's no text. But I am not sure.

Reply:
I just added aria-labels for that purpose.